SPECS/hyperv-daemons/hyperv-daemons.spec

```diff
@@ -140,7 +141,7 @@ install -p -m 0644 %{SOURCE202} %{buildroot}%{_udevrulesdir}/%{udev_prefix}-hype
 mkdir -p %{buildroot}%{_libexecdir}/%{hv_kvp_daemon}
 install -p -m 0755 tools/hv/hv_get_dhcp_info.sh %{buildroot}%{_libexecdir}/%{hv_kvp_daemon}/hv_get_dhcp_info
 install -p -m 0755 tools/hv/hv_get_dns_info.sh %{buildroot}%{_libexecdir}/%{hv_kvp_daemon}/hv_get_dns_info
-install -p -m 0755 tools/hv/hv_set_ifconfig.sh %{buildroot}%{_libexecdir}/%{hv_kvp_daemon}/hv_set_ifconfig
```

Comment:
Wanted to make sure here: how much does the new script differ from the default one? I'm mostly concerned about breaking any existing functionality, so I'd like to better understand what we're changing.

Reply:
Its totally different. That is the core reason for this PR. We cannot use the existing hv_set_ifconfig.sh and it needs to be adapted as per the network manager supported by the distro.

Reply:
I see, thanks for the explanation!